charts/ingress-nginx/values.yaml

```diff
@@ -243,6 +243,17 @@ controller:
     targetCPUUtilizationPercentage: 50
     targetMemoryUtilizationPercentage: 50
 
+  autoscalingTemplate: []
+  # Custom or additional autoscaling metrics
+  # ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics
+  # - type: Pods
+  #   pods:
+  #     metric:
+  #       name: nginx_ingress_controller_nginx_process_requests_total
```

Comment:
@alexandrsemak `nginx_ingress_controller_nginx_process_requests_total` metric is a counter IMO making it not appropriate metrics for HPA; other than being "reset" when container is restarted and pod cycled, just its rate may get reduced when additional replicas get added.

Maybe `nginx_ingress_controller_nginx_process_connections` would be more appropriate. WDYT?

Reply:
@sslavic sorry for confusing by this metric it was provided just for example the main goal was add ability to add any of custom metric. BTW kubernetes use timestamp for calculate averagevalue even with counter metric it will works correct, but you right we can change it to `nginx_ingress_controller_nginx_process_connections` or  wrap `nginx_ingress_controller_nginx_process_requests_total` in `rate`